tubular/nominal.py

```diff
@@ -537,11 +546,20 @@ class MeanResponseTransformer(BaseNominalTransformer, BaseMappingTransformMixin)
         Only created in the mutli-level case. A dictionary of the form level : transformer containing the mean response
         transformers for each level to be encoded against.
 
+    impute_dict: dict
```

Comment:
I think we should avoid using 'impute' as this is typically used in reference to null values and may confuse a user.  `unseen_levels_encoding_dict` might be clearer?

Reply:
Absolutely agree! Replaced impute_dict with unseen_levels_encoding_dict in the commit: [3307aa84](https://github.com/lvgig/tubular/pull/80/commits/3307aa84031d6356f7e42f62d4bb1e41f70f949d)